aws/backing-services/elasticsearch.tf

```diff
@@ -54,12 +54,19 @@ variable "ELASTICSEARCH_PERMITTED_NODES" {
 }
 
 locals {
-  arns = {
+  role_arns = {
     masters = ["${module.kops_metadata.masters_role_arn}"]
     nodes   = ["${module.kops_metadata.nodes_role_arn}"]
     both    = ["${module.kops_metadata.masters_role_arn}", "${module.kops_metadata.nodes_role_arn}"]
     any     = ["*"]
   }
+
+  security_groups = {
+    masters = ["${module.kops_metadata.masters_security_group_id}"]
+    nodes   = ["${module.kops_metadata.nodes_security_group_id}"]
+    both    = ["${module.kops_metadata.masters_security_group_id}", "${module.kops_metadata.nodes_security_group_id}"]
+    any     = ["${module.kops_metadata.masters_security_group_id}", "${module.kops_metadata.nodes_security_group_id}"]
```

Comment:
Why `both` and `any`?

Reply:
`both` and `any` to be consistent with other terraform modules.
But it the case `both` and `any` have the same meaning because no way to grant access from any security group.